OnlineStore.BLL/Services/Interfaces/ICategoryService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using OnlineStore.BLL.DTOs;
+using OnlineStore.DAL.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OnlineStore.BLL.Services.Interfaces
+{
+    public interface ICategoryService
+    {
+        Task<IEnumerable<CategoryDto>> GetAllCategoriesAsync();
+        Task<CategoryDto> GetCategoryByIdAsync(int id);
```

Comment:
you need to throw the token into the methods of the controller, then into the methods of the service and then into the methods of the repositories and into the repositories themselves